collector/rawyaml.go

```diff
@@ -1,5 +1,10 @@
 package collector
 
+/*
+For now at least, we are keeping these as v1beta1 to have some element of regression testing, now that we've flipped
+the "default" to v1.
+*/
+
```

Comment:
placeholder comment regarding some v1beta1 validation

quick reminder - these yaml strings were pulled directly from `oc get pr ... -o yaml` on the staging/prod clusters to most accurately test the time of pipeline runs we have to measure gaps for (with a mix on sequential and parallel tasks within the pipeline run)